client/cmd/client/main.go

```diff
@@ -136,13 +139,22 @@ func loadConfig() (*config.ClientConfig, error) {
 
 	grpcHost := os.Getenv(envGRPCHost)
 	grpcPort := os.Getenv(envGRPCPort)
+	certFile := os.Getenv(envCertFile)
+	certPkFile := os.Getenv(envCertPkFile)
 
 	log.Info("loaded config", "grpc host", grpcHost)
 	log.Info("loaded config", "grpc port", grpcPort)
 
+	log.Info("loaded config", "certificate file", certFile)
+	log.Info("loaded config", "certificate pk", certPkFile)
+
 	return &config.ClientConfig{
```

Comment:
the env variables could be loaded directly from a config.toml file

Reply:
Several reasons:
- for small projects or simple setups, using a single .env file can be straightforward and easy to manage.
- docker Compatibility: .env files are commonly used in Docker containers, and they integrate well with container orchestration tools like Docker Compose and Kubernetes. They allow for easy configuration changes without modifying the application code, which is very useful.

There is already a task created to make it runnable with either .env/configs/flags.